### DISCUSSION TITLE
Default yarn install goes for version 3.4.7, which doesn't exist yet

### DISCUSSION THREAD

**Discussion by allenjhyang** [goatandsheep/react-native-dotenv#404] [Q&A] [answered]
When I first tried to install this library using a simple `yarn add react-native-dotenv`, it installed version 3.4.7. This version doesn't exist yet for this repo, but did exist in the older / now-deprecated repo for react-native-dotenv, which is where I'm guessing it gets 3.4.7 from.

Is that a problem? For now I just specified installing 3.4.2. I haven't gotten things running correctly yet, but I think it's due to Issue #377 - but if I've been installing the wrong repo this whole time, that probably wouldn't help either!

**Comment by goatandsheep** [comment, marked as answer]
Hi that is weird! Can you send me the 3.4.7 from the old repo?

As far as I know, this repo has the tag:
https://github.com/goatandsheep/react-native-dotenv/releases/tag/v3.4.7

I haven't set it as a release yet because I'm waiting for expo to update their Babel.

As far as I know, the old repo doesn't surpass version 0.x
https://github.com/zetachang/react-native-dotenv/tags

I hope you were able to install the latest version by now!
